packages/lesswrong/components/forumEvents/ForumEventPoll.tsx

```diff
@@ -405,147 +537,269 @@ export const ForumEventPoll = ({postId, hideViewResults, classes}: {
+                    </div>
+                  )}
+                  {cluster.votes
+                    .slice(-MAX_STACK_IMAGES)
+                    .map((vote) => (
+                      <div
+                        key={vote.user._id}
+                        className={classes.voteCircle}
+                      >
                         <LWTooltip
-                          key={vote.user._id}
-                          title={<div className={classes.voteTooltipBody}>{vote.user.displayName}</div>}
+                          title={
+                            <div className={classes.voteTooltipBody}>
+                              {vote.user.displayName}
+                            </div>
+                          }
                         >
                           <UsersProfileImage
                             user={vote.user}
+                            // The actual size gets overridden by the styles above. This
+                            // is still needed to get the right resolution from Cloudinary
                             size={USER_IMAGE_SIZE}
-                            className={classNames(classes.userImage, classes.userImageBoxShadow)}
+                            className={classNames(
+                              classes.userImage,
+                              classes.userResultsImage
+                            )}
                           />
                         </LWTooltip>
-                      ))}
-                    </div>
-                  }
-                  placement="bottom"
-                  clickable
-                  className={classes.voteCluster}
+                      </div>
+                    ))}
+                </div>
+              ))}
+            </div>
+            <div
+              className={classes.sliderLine}
+              ref={sliderRef}
+            >
+              {/* Ticks */}
+              <div className={classes.ticksContainer}>
+                {ticks.map((tickIndex) => (
+                  <div
+                    key={tickIndex}
+                    ref={(el) => (tickRefs.current[tickIndex] = el)}
+                    className={classNames(
+                      classes.tick,
+                      isDragging.current && classes.tickDragging,
+                      tickIndex === Math.floor(NUM_TICKS / 2) &&
+                        classes.centralTick
+                    )}
+                  />
+                ))}
+                {/* User Vote */}
+                <div
+                  className={classNames(
+                    classes.userVote,
+                    classes.currentUserVote,
+                    isDragging.current && classes.currentUserVoteDragging,
+                    hasVoted && classes.currentUserVoteActive
+                  )}
+                  onPointerDown={startDragVote}
+                  style={{
+                    left: `${votePos}%`,
+                    transform: "translateX(-50%)",
+                  }}
```

Comment:
I noticed that the vote circle gets cut off at the edges of the slider. Not blocking, but if you can do a quick z-index thing it would be good to fix.

<img width="144" alt="Screenshot 2024-10-02 at 8 42 17 PM" src="https://github.com/user-attachments/assets/34714710-3cc9-48ef-a897-6b01592fad75">

Reply:
Thanks, it just needed a bit of padding